tests/functional/transaction_timeout.ts

```diff
@@ -99,8 +102,11 @@ describe('Transaction timout will reset the transaction id for the claim', async
     expected: true,
   })
 
-  await btcdClientB.generate(blockchainSettings.MAXIMUM_TRANSACTION_AGE_IN_BLOCKS + 1)
-  await btcdClientA.setNetworkActive(true)
+  await bitcoinCoreClientB.generate(blockchainSettings.MAXIMUM_TRANSACTION_AGE_IN_BLOCKS + 1)
+  const targetHeight = 102 + blockchainSettings.MAXIMUM_TRANSACTION_AGE_IN_BLOCKS
```

Comment:
Can we move this magic number to a constant or infer it from an already declared one?